index.d.ts

```diff
@@ -1156,6 +1166,14 @@ declare namespace algoliasearchHelper {
      * total number of hits of this query on the index
      */
     nbHits: number;
+    /**
+     * total number of hits from smart sort
```

Comment:
can we use the same descriptions as in the client? https://github.com/algolia/algoliasearch-client-javascript/pull/1251

Reply:
Oh nice, I didn't know they were already there. https://github.com/algolia/algoliasearch-helper-js/pull/809/commits/107374a11b059135098734586cef082742bd256c